values.sample.yaml

```diff
@@ -25,11 +25,36 @@ enableLogCollection: false
 # This value will be provided to you by Tonic and will allow you to authenticate against our private docker repository.
 dockerConfigAuth: <docker-config-auth>
 
+# service account for tonic
+serviceAccount:
+  create: true
```

Comment:
Should this default to true or false?

Reply:
it can be false, it will just use the default service account for its namespace then.